src/include/daos_errno.h

```diff
@@ -161,7 +161,10 @@ extern "C" {
 	       incompatible user or group permissions)			\
 	/** Fatal (non-retry-able) transport layer mercury error */	\
 	ACTION(DER_HG_FATAL,		(DER_ERR_GURT_BASE + 45),	\
-	       Fatal transport layer mercury error)
+	       Fatal transport layer mercury error)			\
+	/** Failed to send rpc */                                       \
+	ACTION(DER_HG_SEND_FAILED,      (DER_ERR_GURT_BASE + 46),       \
+		Failed to send RPC)
```

Comment:
Can we just remove this? Wouldn't that cause compatibility issues?

Reply:
What do you want removed?  I think the added lines of code are needed.

Reply:
Never mind. The GitHub diff was showing that the above DER_HG_FATAL was removed entirely. But now it looks like you just added a new one, which is okay.